### PR TITLE
[FEAT] Implement AboutController

### PR DESCRIPTION
**Pull request by kingsqd**
In this PR I am implementing the UI for the More Controller

The More tab will contain descriptions about the project and how to get involved.  

TODO: add links to buttons.  Add real text for descriptions.

Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-08 at 03 30 49

### CODE REVIEW

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+/// The UI for More
+final class MoreView: UIView {
+    
+    private lazy var aboutCard: UIView = {
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreView.swift:30]: Can we clean up the creation of this view into a new custom view?

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+            titleLabel.text = Strings.MoreHistory.title
+        case .developer:
+            titleLabel.text = Strings.GetInvolvedDev.title
+        case .designer:
+            titleLabel.text = Strings.GetInvolvedDesign.title
+        }
+        return titleLabel
+    }
+    
+    private func descriptionLable(_ section: section) -> UILabel {
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreView.swift:177]:
```suggestion
    private func descriptionLabel(_ section: section) -> UILabel {
```

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 3]
```diff
@@ -24,16 +24,19 @@
 
 import UIKit
 
-class AboutController: UIViewController {
-    private let aboutView = AboutView()
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreController.swift:28]: Let's keep this here but rename it
```
private let moreView = MoreView()
```

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 4]
```diff
@@ -24,16 +24,19 @@
...
     }
     
     required init?(coder: NSCoder) { fatalError("This should not be called") }
-
-    override func loadView() {
-        self.view = aboutView
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let moreView = MoreView(frame: self.view.frame)
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreController.swift:37]:
```suggestion
```

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 5]
```diff
@@ -24,16 +24,19 @@
...
+        self.view.addSubview(moreView)
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreController.swift:38]:
```suggestion
```

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 6]
```diff
@@ -24,16 +24,19 @@
...
 
     required init() {
         super.init(nibName: nil, bundle: nil)
     }
     
     required init?(coder: NSCoder) { fatalError("This should not be called") }
-
-    override func loadView() {
-        self.view = aboutView
+
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Controller/More/MoreController.swift:34]: If we use `loadView` we dont have to have our UI as a subview
```suggestion
    override func loadView() {
    view = moreView
```

**Comment by kingsqd** [comment]
@Kilo-Loco  thanks for the review ! Seems like the asset I was using for the logo got removed, any idea how we can add that back ?

**Comment by Kilo-Loco** [comment]
> @Kilo-Loco thanks for the review ! Seems like the asset I was using for the logo got removed, any idea how we can add that back ?

One of the recent PRs have implemented assets as part of SwiftGen. What this means for you is that instead of using an image by its String name, you would create your image like this
```
imageView.image = UIImage(asset: STNImage.whiteLogo)
```

**Comment by kingsqd** [comment]
@Kilo-Loco this is a different icon. This new one is much larger. What is the best approach for resizing ?

**Pull request merged**